<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParticipantsPane</class>
 <widget class="QWidget" name="ParticipantsPane">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>506</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="participantsLabel">
     <property name="text">
      <string>Permission</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QTreeView" name="m_treeview">
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="QToolButton" name="m_upToolBtn">
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="m_downToolBtn">
         <property name="text">
          <string notr="true">...</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="connectInfoLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="m_promoteAction">
   <property name="icon">
    <iconset theme="go-top" resource="../../../rolisteam.qrc">
     <normaloff>:/resources/rolistheme/go-top.png</normaloff>:/resources/rolistheme/go-top.png</iconset>
   </property>
   <property name="text">
    <string>Promote</string>
   </property>
  </action>
  <action name="m_demoteAction">
   <property name="icon">
    <iconset theme="go-bottom">
     <normaloff>:/resources/images/go-down.png</normaloff>:/resources/images/go-down.png</iconset>
   </property>
   <property name="text">
    <string>Demote</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../rolisteam.qrc"/>
 </resources>
 <connections/>
</ui>
